gneiss/balances.py

```diff
@@ -0,0 +1,115 @@
+from __future__ import division
+import numpy as np
+from skbio.stats.composition import clr_inv
+from collections import OrderedDict
+
+
+def _balance_basis(tree_node):
+    """ Helper method for calculating balance basis
+    """
+
+    n_tips = sum([n.is_tip() for n in tree_node.traverse()])
```

Comment:
As large as we can visualize them.  From prior experience, no more than 4000 tips - it starts to become very difficult to visualize anything past that.